### PR TITLE
Add Third Reality as integration

### PR DESCRIPTION
**Pull request by balloob**
## Proposed change

This adds a Third Reality integration so they are visible on the website. 

Preview of the Third Reality page https://deploy-preview-24555--home-assistant-docs.netlify.app/integrations/third_reality

https://www.home-assistant.io/blog/2022/10/13/third-reality-partner/

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/third_reality.markdown [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  - Binary Sensor
+  - Button
+ha_domain: third_reality
+ha_integration_type: integration
+works_with:
+  - zigbee
+ha_platforms:
+  - binary_sensor
+  - button
+  - sensor
```
Comment (MartinHjelmare) [inline source/_integrations/third_reality.markdown:20]: The platforms and categories don't match.

File: source/_integrations/third_reality.markdown [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: Third Reality
+description: Connect and control your Third Reality Zigbee devices using the Zigbee integration
+ha_release: '2021.10'
```
Comment (MartinHjelmare) [inline source/_integrations/third_reality.markdown]: Which release supports this?

Comment (balloob) [reply source/_integrations/third_reality.markdown]: The brand has been added in the current release.

Comment (KevinCathcart) [reply source/_integrations/third_reality.markdown]: Current release would be 2022.10, not 2021.10, right?

Comment (balloob) [reply source/_integrations/third_reality.markdown]: 👀 
```suggestion
ha_release: '2022.10'
```

File: source/_includes/asides/component_navigation.html [hunk 3]
```diff
@@ -6,36 +6,44 @@
...
-      {{ percentage | remove: ".0" }}%</a> of the active installations.
-    {% endif %}
-
-    {%- if page.ha_iot_class %}
-      Its IoT class is <a href='/blog/2016/02/12/classifying-the-internet-of-things/#classifiers'>{{ page.ha_iot_class }}</a>
-    {%- endif -%}
+    {%- if page.ha_brand -%}
+      The {{ page.name | default: page.title }} brand was introduced in Home Assistant {{ page.ha_release | default: "unknown" }}.
+    {%- else -%}
+      The {{ page.name | default: page.title }} {{ page.ha_integration_type | default: "integration" }} was introduced in Home Assistant {{ page.ha_release | default: "unknown" }},
```
Comment (balloob) [inline source/_includes/asides/component_navigation.html:25]: This has just been indented. [See diff without whitespace changes](https://github.com/home-assistant/home-assistant.io/pull/24555/files?w=1)

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**